examples/chip-tool/README.md

```diff
@@ -159,23 +159,25 @@ The client will send a single command packet and then exit.
 2. Add Group to device
 
     ```
+    $ chip-tool groups add-group GroupId GroupName node-id/group-id endpoint-id
```

Comment:
```suggestion
    $ chip-tool groups add-group GroupId GroupName node-id endpoint-id
```
Because I doubt you'll actually want to add groups as a group command and it's confusing to have GroupId and group-id here.

Similar for the other bits?

Reply:
You're right that is confusing. Cleared it up to only show node-id in that field.